integration/spark-common/src/main/scala/org/apache/carbondata/spark/util/DataTypeConverterUtil.scala

```diff
@@ -77,4 +77,40 @@ object DataTypeConverterUtil {
       case DataType.STRUCT => "struct"
     }
   }
+
+  /**
+   * convert from wrapper to external data type
+   *
+   * @param dataType
+   * @return
+   */
+  def convertToThriftDataType(dataType: String): org.apache.carbondata.format.DataType = {
```

Comment:
There is no direct string to thrift datatype conversion. When creating table, we first convert string to Wrapper datatype and from wrapper datatype, we convert into thrift datatype. This method is required for alter table, as we r directly converting string to thrift type in alter table change datatype